src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -882,6 +886,21 @@ private void ExecuteFindReferences(ITextBuffer subjectBuffer, IContentType conte
                 lastHandler: executeNextCommandTarget);
         }
 
+        protected virtual void SyncClassView(SnapshotPoint caretPosition)
+        {
+        }
+
+        void ExecuteSyncClassView(ITextBuffer subjectBuffer, IContentType contentType, Action executeNextCommandTarget)
```

Comment:
Well, it's a type for the command args, a type for the command handler, etc.  I'll do it though.